dockerfiles/base/scripts/base/commands/cmd_stop.sh

```diff
@@ -29,13 +29,23 @@ cmd_stop() {
   FORCE_STOP=false
   if [[ "$@" == *"--skip:graceful"* ]]; then
   	FORCE_STOP=true
+  elif local_repo; then
+    warning "Development mode [skip graceful stop]"
```

Comment:
In the issue ppl were asking for local repo mode
I'm also ok for local_assembly. I will add